manager/allocator/network.go

```diff
@@ -383,11 +393,12 @@ func (a *Allocator) doNetworkAlloc(ctx context.Context, ev events.Event) {
 	case state.EventCommit:
 		a.procTasksNetwork(ctx, false)
 
-		if time.Since(nc.lastRetry) > retryInterval {
+		if time.Since(nc.lastRetry) > retryInterval || nc.somethingWasDeallocated {
```

Comment:
`somethingWasDeallocated` is only set during an event. These events are always followed by `EventCommit`.